frontend/src/models/analyses.js

```diff
@@ -308,6 +308,90 @@ const annotationRenderingTemporary = [
       },
     ],
   },
+  {
+    'type': 'section',
+    'class': '',
+    'header': 'Chromosomal Localization',
+    'anchor': 'Chromosomal_Localization',
+    'header_datasets': [],
+    'props': {
+      'attachmentDataset': 'Chromosomal_Localization',
+      'genomicAttachmentType': 'variant',
+    },
+    'rows': [{
+      'class': '',
+      'datasets': [
+        {
+          'dataset': 'Chromosomal_Localization',
+          'genomicType': 'variant',
+          'type': 'images-dataset',
+        },
+      ],
+    }],
+  },
+  {
+    'type': 'section',
+    'class': '',
+    'header': 'Secondary Structure',
+    'anchor': 'Secondary_Structure',
+    'header_datasets': [],
+    'props': {
+      'attachmentDataset': 'Secondary Structure',
+      'genomicAttachmentType': 'variant',
+    },
+    'rows': [{
+      'class': '',
+      'datasets': [
+        {
+          'dataset': 'Secondary_Structure',
+          'genomicType': 'variant',
+          'type': 'images-dataset',
+        },
+      ],
+    }],
+  },
+  {
+    'type': 'section',
+    'class': '',
+    'header': 'Casual Variant In This Locus (ClinVar)',
+    'anchor': 'Casual_Variant',
+    'header_datasets': [],
+    'props': {
+      'attachmentDataset': 'Casual Variant In This Locus (ClinVar)',
+      'genomicAttachmentType': 'variant',
+    },
+    'rows': [{
+      'class': '',
+      'datasets': [
+        {
+          'dataset': 'Casual Variant In This Locus (ClinVar)',
```

Comment:
Switching the spaces to to dashes and removing the '('s from teh dataset name will make it easier to query in the future.  We aren't full on supporting the system configuring/setting these names which is why we haven't had the opportunity to document this.  Its a work in progress and still figuring out things. (and it will need line 360 to match whatever you update it to).

Reply:
Sounds good. made this change